addon/object/computed.js

```diff
@@ -1,10 +1,11 @@
 import Ember from 'ember';
-
 import {
   decoratedPropertyWithRequiredParams,
   decoratedPropertyWithOptionalCallback
 } from '../utils/decorator-macros';
 
+const { assert } = Ember;
```

Comment:
Sorry for the issues in the underlying lib, we definitely want to use `import { assert } from '@ember/debug';` though. See https://github.com/rwjblue/ember-decorators/pull/148#issuecomment-318432401 for a work around.